tests/e2e/utils/auto-rds-s3-cleanup.py

```diff
@@ -0,0 +1,89 @@
+import argparse
+from time import sleep
+import boto3
+
+from utils import get_rds_client, get_s3_client, get_secrets_manager_client
+
+
+def main():
+    metadata_dict = {}
+    with open("auto-rds-s3-setup-metadata", "r") as setup_metadata:
```

Comment:
I think we should write to yaml file instead having to write extra code for it

We have read_yaml_file and write_yaml_file in utils
https://github.com/awslabs/kubeflow-manifests/blob/8c770276d98e3236d4f774e0cb459b68467656e1/tests/e2e/utils/utils.py#L205-L214